_includes/footer.html

```diff
@@ -1,14 +1,20 @@
 <footer class="footer">
   <div class="container">
     <div class="columns">
-      <div class="column">
+      
+      <div class="column" style="text-align: center;">
+        <img src="https://avatars.githubusercontent.com/u/25749259?s=150&amp;v=4" alt="Drawpile Logo"><br>
```

Comment:
Don't hotlink a GitHub implementation detail. If anything, put the logo into `assets/img` instead.

Something about this also breaks the responsiveness, not sure if it's the logo or the centered text. If you narrow the page, it wonks out at one point and you get weird line breaks.

![image](https://github.com/drawpile/drawpile.github.io/assets/13625824/37cbe9b3-6aba-44ba-9fd4-8fb8748010e0)